turbinia/processors/google_cloud.py

```diff
@@ -24,292 +24,13 @@
 
 from turbinia import config
 from turbinia import TurbiniaException
+from turbinia.lib import google_cloud
```

Comment:
Maybe this instead:
from turbinia.lib.google_cloud import GoogleCloudProject

Reply:
Yeah, that's probably better.  Done.